eth/chains/base.py

```diff
@@ -577,13 +585,24 @@ def import_block(self, block: BaseBlock, perform_validation: bool=True) -> BaseB
             ensure_imported_block_unchanged(imported_block, block)
             self.validate_block(imported_block)
 
-        self.chaindb.persist_block(imported_block)
+        new_canonical_header_hashes,\
```

Comment:
*formatting nitpick*  we don't have any of these style line continuations and I'd like to avoid adding any.  Can you update this to:

```python
(
    new_canonical_header_hashes,
    orphaned_header_hashes,
) = self.chaindb.persist_block(imported_block)
```